script/find_record.py

```diff
@@ -0,0 +1,36 @@
+import json 
+from sys import argv
+
+
+script, JSON_FILE, user_name  = argv
+
+
+unflat_persons = json.load(open(JSON_FILE, encoding='utf-8'))
+ 
+
+def find_users_by_name(first_name):
+  list_of_persons = unflat_persons['results']
+  user_list = []
+  for dict_of_person in list_of_persons:   
+    gender = dict_of_person['gender']
+    title = dict_of_person['name']['title']
+    first_name = dict_of_person['name']['first']
+    last_name = dict_of_person['name']['last']
+    if first_name == user_name:
+      user_list.append((gender, title, first_name, last_name))
+  return user_list
+
+
+def print_user_names(users):  
```

Comment:
This function doesn't print user names. So it would be better to change its name according to what it is doing.
Also print information with better formatting then just putting python object to print function.

Reply:
Using the "Columnar" library?

Reply:
No need here. Do it yourself :-)

Reply:
It was about that?

Reply:
I don't see any new code.
How the output of this function looks like?

Reply:
I changed in line 20, the output looks like this:
`['female Miss Sophia Roy', 'female Mrs Sophia Patel']`

Reply:
This is not good formatting.
Ideally the output should looks like this:
```
# | Sex      | Title  | First name | Last name
---------------------------------------------
1 | female | Miss | Sophia        | Roy
```

This formatting should be done in function print